### ISSUE TITLE
FR: Can figure the padding of a text

### ISSUE THREAD

**Issue by jzhang-brex** [zsviczian/obsidian-excalidraw-plugin#1115]
**Is your feature request related to a problem? Please describe.**

![CleanShot 2023-04-27 at 07 12 32](https://user-images.githubusercontent.com/1160963/234889993-ea7cacff-d4d1-487a-b40a-59ac6493a441.png)

Here is an example text box. When the text is left-align, it's not possible to configure the margin. It will be best to be able to configure horizontal and vertical padding.

This is less of a problem when the text is centered. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Esthetech** [comment]
@zsviczian did you add this custom property in one of the subsequent releases?

The lack of this padding right now makes me not wanna use Excalidraw in presentations because it looks ugly to me and makes reading hard and unpleasant.

**Comment by zsviczian** [comment]
@Esthetech I tend to agree, however this needs to be changed in the Excalidraw component not in Obsidian. if I change it in Obsidian only, that will break compatibility which is an issue for many, including myself. I have a PR open with the Excalidraw team for this and some other related features, unfortunately it is not getting traction. Maybe you can raise this as an issue on excalidraw.com and reference the open PR as a solution. Maybe it will grab attention:

https://github.com/excalidraw/excalidraw/pull/10001

**Comment by dimateos** [comment]
I will check it out as this bothers me a bit too!

If it fails to get traction, you could consider simply altering the hard-coded constant:
https://github.com/zsviczian/excalidraw/blob/dc40f91957480c8f4b00d86345b0632b671bb4e7/packages/common/src/constants.ts#L389

- 5 px just feels like too little for top/left margin

**Comment by zsviczian** [comment]
@dimateos, I don't think you got my point earlier regarding compatibility. Changing it is simple; the issue is backwards compatibility with the millions of drawings people have already made and ongoing compatibility with Excalidraw.com. Changing the constant is not an option. Adding an element property and defaulting it to the old constant is, but only if excalidraw.com follows this else, whenever someone copies a drawing from Obsidian to Excalidraw it is going to look different, with text containers growing, etc. That is simply not an option. This is an issue to press with Excalidraw.com, I am happy to work on the PR, but only if it gets traction, else I am just spinning my wheels without value.
